bin/runSmokeTests.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
-set -ex
 
-docker-compose -f docker-compose.tests.yml run --name "ccd-case-management-protractor" --rm -u `id -u $USER` protractor test:smokeTests
+##set -ex
+##docker-compose -f docker-compose.tests.yml run --name "ccd-case-management-protractor" --rm -u `id -u $USER` protractor test:smokeTests
```

Comment:
If you're dropping the docker approach then remove it please.

Reply:
Will remove them. Cheers.